docker-compose.yml

```
@@ -0,0 +1,34 @@
+services:
+  postgres:
+    image: postgres:latest
+    restart: always
+    shm_size: 128mb
+    container_name: postgres
+    environment:
+      POSTGRES_USER: ${DATABASE_USER}
+      POSTGRES_PASSWORD: ${DATABASE_PASSWORD}
+      POSTGRES_DB: ${DATABASE_NAME}
+    ports:
+      - "5433:5432"
+       
+  frontend:
+    build:
+      context: ./frontend
+      dockerfile: Dockerfile.frontend.dev
+    ports:
+      - "3000:3000"
+
+  backend:
+    build:
+      context: ./backend
+      dockerfile: Dockerfile.backend.dev
+    depends_on:
+      - postgres
+    ports:
+      - "8000:8000"
+    volumes:
+      - ./backend:/app/backend
+    env_file:
+      - ./.env
+    environment:
+      - DJANGO_SETTINGS_MODULE=djangoindia.settings.local
```

lets move it to top.
```suggestion
  djangoindia-backend: &djangoindia-backend
    build:
      context: ./backend
      dockerfile: Dockerfile.backend.dev
    container_name: djangoindia-backend
    depends_on:
      postgres:
        condition: service_healthy
    ports:
      - "8000:8000"
    volumes:
      - ./backend:/app/backend
    env_file:
      - .env
    environment:
      - DJANGO_SETTINGS_MODULE=djangoindia.settings.local
  djangoindia-worker:
    <<: *djangoindia-backend
    container_name: djangoindia-worker
    command: celery -A backend/djangoindia worker -l info 
    ports: []
```